app/dashboard/utils.py

```diff
@@ -284,7 +284,7 @@ def ipfs_cat_requests(key):
         return None, 500
 
 
-def get_web3(network, sockets=False):
+def get_web3(network, sockets=False, is_polygon=False):
```

Comment:
would it make sense to network_name or something generic and the value could be `polygon`?

Reply:
done